### PR TITLE
fix: fixes for data migration

### PR DESCRIPTION
**Pull request by IgnatG**
### Changelog:
- fixed the mandatory criteria versioning
- exclude `securityDetails` from being migrated with the azure files
- sanitised the filenames to be used for sharepoint

### CODE REVIEW

File: e2e-tests/portal/cypress/integration/journeys/login/reset-password.spec.js [hunk 1]
```diff
@@ -20,19 +20,20 @@ context('Reset password', () => {
       resetPassword.resetPasswordError().should('exist');
     });
 
-    it('should redirect to login page on successful request for reset password', () => {
-      resetPassword.emailInput().type('test_no_notify@ukexportfinance.gov.uk');
-      resetPassword.submit().click();
+    // TODO: re-enable the tests once the user is added back in the mock data loader
```
Comment (Zainzzkk) [inline e2e-tests/portal/cypress/integration/journeys/login/reset-password.spec.js:23]: is there a ticket number for this todo?

File: utils/data-migration/gef/map-facilities.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ const mapCoverEndDate = (coverStartDate, exposurePeriod) => {
     months: exposurePeriod,
   });
 
-  return getUnixTime(date);
+  return new Date(getUnixTime(date));
```
Comment (abhi-markan) [inline utils/data-migration/gef/map-facilities.js:71]: We are not storing date as EPOCH?

**Review by abhi-markan — COMMENTED** [review]
A minor clarification please.

File: utils/data-migration/gef/map-facilities.js (hunk 2 above)
Comment (IgnatG) [reply utils/data-migration/gef/map-facilities.js:71]: unfortunately not for cover end date.

Comment (abhi-markan) [reply utils/data-migration/gef/map-facilities.js:71]: Ah okay, can we store it as EPOCH or is it going to be a big job to do so and out of scope for this PR?

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**